### ISSUE TITLE
Clicking on "manual sync" does not initiate the one-time sync.

### ISSUE THREAD

**Issue by santosh-pingle** [google/android-fhir#2492]
**Describe the bug**
Clicking on manual sync does not trigger the one time sync.

**Component**
reference app,

**To Reproduce**
1. Open demo app
2. Create new patient 
3. Click on manual sync
4. See, manual is not triggered

**Expected behavior**
It should trigger the manual sync, and sync the patient.

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: Pixel2a emulator
 - Android version: 12

**Additional context**

**Would you like to work on the issue?**
Please state if this issue should be assigned to you or who you think could help to solve this issue.

**Comment by santosh-pingle** [comment]
This might have caused the issue #2348

**Comment by santosh-pingle** [comment]
@@hoangchungk53qx1 can you please take look at it?

**Comment by hoangchungk53qx1** [comment]
> @@hoangchungk53qx1 can you please take look at it?

i'll check it again tomorrow

**Comment by hoangchungk53qx1** [comment]
i just checked, everything is still working
Let me try again with your device

**Comment by hoangchungk53qx1** [comment]
@santosh-pingle i see it, i create MR (https://github.com/google/android-fhir/pull/2507)
